rest/route/service.go

```diff
@@ -165,6 +165,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/subscriptions").Version(2).Post().Wrap(checkUser).RouteHandler(makeSetSubscription(sc))
 	app.AddRoute("/tasks/{task_id}").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeGetTaskRoute(sc))
 	app.AddRoute("/tasks/{task_id}").Version(2).Patch().Wrap(checkUser, addProject, editTasks).RouteHandler(makeModifyTaskRoute(sc))
+	app.AddRoute("/task/{task_id}/annotation").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeFetchAnnotationsByVersion(sc))
```

Comment:
wrong function here, you want makeFetchAnnotationsByTask
also nit: change to "annotations" to be consistent with the other routes, and with the list format.